### ISSUE TITLE
Move change point detection to new section

### ISSUE THREAD

**Issue by robjhyndman** [cran-task-views/TimeSeries#20]
There are now so many change point detection packages, that it would be better for them to have their own section, rather than being part of Forecasting and Univariate modelling.

As well as the ones already listed, new ones include
- [ ] detectR
- [ ] rid

**Comment by rkillick** [comment]
Agreed, there has been quite an explosion recently so I will take a look at them and add a bit more explanation on the differences.
